src/app/components/job/details/job-configuration.html

```diff
@@ -38,8 +38,8 @@
                 <span *ngIf="job?.executionInfo?.endTime">Completed in: {{ executionInfo.runtime }}</span>
             </div>
 
-            <bl-text-property label="Start time" [value]="executionInfo.startTime"></bl-text-property>
-            <bl-text-property label="End time" [value]="executionInfo.endTime"></bl-text-property>
+            <bl-text-property label="Start time" [value]="job?.executionInfo.startTime"></bl-text-property>
```

Comment:
why here and not below then?

Reply:
start and end time aren't on the JobExecutionInfoDecorator. Might be better  to just add them there Ill do that instead

Reply:
Already followed this paradigm for task/other execution times, but really we should just have all the information on the decorator

Reply:
oh I see, actually the decorators are coming from the portal and I don't think they are a good idea to use with angular. Better have a minimal component to display the data a certain way. The decorator don't handle data change very well. I'd recommmened you try to slowly get rid of them. I removed quite a few already

Reply:
There is actually a `bl-date-property` should use that instead. This won't work with the timezone feature

Reply:
AH i remember you mentioning that! I'll revert and change to date prop